src/components/sbb-datepicker-previous-day/sbb-datepicker-previous-day.tsx

```diff
@@ -24,88 +12,97 @@ import {
   HandlerRepository,
   actionElementHandlerAspect,
   languageChangeHandlerAspect,
+  ConnectedAbortController,
 } from '../../global/eventing';
 import { isValidAttribute, toggleDatasetEntry } from '../../global/dom';
+import { CSSResult, html, LitElement, TemplateResult, PropertyValues } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { SbbDatepicker } from '../sbb-datepicker/index';
+import { setAttribute, setAttributes } from '../../global/dom';
+import Style from './sbb-datepicker-previous-day.scss?lit&inline';
+
+import '../sbb-icon';
+import '../sbb-datepicker';
```

Comment:
```suggestion
```
The datepicker is not an internal dependency of this component (you would need the import only if you were using it in the template)